src/Dialogs/SyncWarningDialog.vala

```diff
@@ -73,75 +79,58 @@ public class Noise.SyncWarningDialog : Gtk.Window {
         info.set_markup (info_text);
 
         // be a bit explicit to make translations better
-        string title_text = "";
-        if (to_remove.size > 1) {
-            title_text = _("Sync will remove %i items from %s").printf (to_remove.size, d.getDisplayName ());
-        }
-        else {
-            title_text = _("Sync will remove 1 item from %s").printf (d.getDisplayName ());
-        }
+        string title_text = to_remove.size > 1
+            ? _("Sync will remove %i items from %s").printf (to_remove.size, d.getDisplayName ())
+            : _("Sync will remove 1 item from %s").printf (d.getDisplayName ());
 
```

Comment:
I think the message will just be `Sync will remove 0 item from [Device Name Here]`.